flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -187,8 +186,6 @@ run_test "TPC-DS end-to-end test (Blink planner)" "$END_TO_END_DIR/test-scripts/
 
 run_test "Heavy deployment end-to-end test" "$END_TO_END_DIR/test-scripts/test_heavy_deployment.sh" "skip_check_exceptions"
 
-run_test "ConnectedComponents iterations with high parallelism end-to-end test" "$END_TO_END_DIR/test-scripts/test_high_parallelism_iterations.sh 25"
```

Comment:
I would refrain from excluding this test until we have a confirmation that this is indeed java 11 related and some numbers of how reliably it fails.

Reply:
I can not remember having seen this failure on jdk8 ... In the last 48 hours, I have changed my policy to creating tickets for each test failure I see.

How many successful executions on `master` is proof enough that the test is stable on jdk8? (I propose 10)
How long should I keep the JIRA ticket open until we believe it is a jdk11 only issue? (I propose a week)

If we are not excluding this test for the JDK11 build, we won't have e2e coverage for jdk11 for everything that comes after.

Reply:
This test has always been _incredibly_ flaky, that's why I advise against rash calls.

Trivial solution for preventing other tests from running in case of failures is to move it to the end of the script.

Reply:
naturally if you can show that it is reproducible on java 11 then we can of course disable it.

Reply:
Looks like I am able to fix the test on JDK11: https://issues.apache.org/jira/browse/FLINK-16417